components/avatar/avatar.html

```diff
@@ -22,31 +22,57 @@
                     <li><a href="">Documentation</a></li>
                 </ul>
             </nav>
-            <a href="#">About Me</a>
+            <a href="#">Examples</a>
         </header>
         <nav class="side-navbar">
             <ul class="side-ul">
                 <h1>Components</h1>
-                <li class="side-li"><a href="">Nav 1</a></li>
-                <li class="side-li"><a href="">Nav 2</a></li>
-                <li class="side-li"><a href="">Nav 3</a></li>
-                <li class="side-li"><a href="">Nav 3</a></li>
-                <li class="side-li"><a href="">Nav 1</a></li>
-                <li class="side-li"><a href="">Nav 2</a></li>
-                <li class="side-li"><a href="">Nav 3</a></li>
+                <li class="side-li"><a href="">Avatar</a></li>
+                <li class="side-li"><a href="">Alert</a></li>
+                <li class="side-li"><a href="">Badge</a></li>
+                <li class="side-li"><a href="">Button</a></li>
+                <li class="side-li"><a href="">Card</a></li>
+                <li class="side-li"><a href="">Image</a></li>
+                <li class="side-li"><a href="">Input</a></li>
                 <li class="side-li"><a href="">Nav 3</a></li>
                 <li class="side-li"><a href="">Nav 1</a></li>
                 <li class="side-li"><a href="">Nav 2</a></li>
                 <li class="side-li"><a href="">Nav 3</a></li>
                 <li class="side-li"><a href="">Nav 3</a></li>
             </ul>
         </nav>
-        <article class="content">
-            <h1>Main article area</h1>
-            <p>In this layout, we display the areas in source order for any screen less that 500 pixels wide. We go to a
-                two column layout, and then to a three column layout by redefining the grid, and the placement of items
-                on the grid.</p>
-        </article>
+        <section class="main-section">
+            <div class="section-heading">Avatar</div>
+            <div class="section-subhead">Avatar has many use cases of different sizes and types for eg.profile pictures
+            </div>
+            <p class="section-para">In the below example we have used image in an avatar form. Use classes avatar-xs
+                ,avatar-sm,avatar-md and avatar-lg .</p>
+            <div class="component-example flex-row flex-align-center flex-gap1 first">
+                <div><img class="avatar avatar-xs" src="face.JPG" alt=""></div>
```

Comment:
alt tags should not be empty and also the div could be removed here...

Reply:
sure would fix it